includes/cli-verbosity-no-effect.md

```diff
@@ -0,0 +1,3 @@
+- **`-v|--verbosity <LEVEL>`**
+
+  Although `-v|--verbosity` is accepted as a valid option, the verbosity setting has no effect on the command's output.
```

Comment:
I'm not sure we want to include this in the docs, as it could very well change later.

Reply:
Got it, I'll revert this change.